changelog/C/Crown.md

```diff
@@ -0,0 +1,10 @@
+# What Has Changed In the “Crown” Article
+
+## Major changes
+The following words or expressions were changed because they seemed too complicated to the general understanding of the target audience:
+- a narrow band of silk studded with jewels --> a narrow band of silk covered with jewels
+- wreath --> crown 
```

Comment:
A handful of other articles use the word wreath as well, perhaps if this change becomes systematic we can add it to the replacement table in the General Style Guide.md